### DISCUSSION TITLE
Is there a way to make the carousel vertical?

### DISCUSSION THREAD

**Discussion by wlsquid** [fancyapps/ui#77] [Q&A]
I have been fiddling with the carousel app and I am wondering if it is possible to make it vertical? You can make it vertical with css but it 'slides' the wrong way (start <-> end). 

also is it possible to use the `Image : { }` options in carousel?

**Comment by fancyapps** [comment]
Hi,

1) There is currently no option for that, but it is planned.
2) No, but what do you need it for? If you want to have pan/zoom functionality, then you would have to create Panzoom instance for each slide, like in this demo - https://fancyapps.com/playground/vs

**Reply by chadboettcher** [reply]
Hi,

What is the timeline for the vertical carousel?  I guess I should clarify: when you say that a vertical carousel is planned, do you mean the thumbnail carousel like the one demoed at https://fancyapps.com/playground/vu or are you referring to the fancybox pop-up gallery?

I started with https://fancyapps.com/playground/vu and have modified it to show the thumbnail carousel vertically on the left.  It's mostly working but I had to override `carousel__track` `transform` with `transform: none !important;`. The main issue I'm having is clicking on thumbnails sometimes requires multiple clicks before it will switch the main image out.  The fancybox pop-up gallery implementation when I click the main image is fine as is (thumbnails across the bottom) and works as expected.

Thanks.

**Reply by chadboettcher** [reply]
As it turns out, I was missing the carousel class on the thumbnail container so that fixed some of the wonkiness.  However, in order to get to really work, I had to to add the `data-index` attributes to both carousels as I rendered them. After that, it worked as expected.

I'd still like to get clarification as to which vertical carousel is being planned.

Thanks.

**Reply by fancyapps** [reply]
Sorry, I do not understand what to clarify. Vertical carousel is carousel that you can drag ... vertically.
